### PR TITLE
feat: Fixes #91 add README.md

### LINKED ISSUE #91: Add README to generated repos
Include:
- One line description
- Basic usage
  - make install
  - make run
  - create config/samples
  - build companion CLI
- next steps for further dev

To add later: link to API docs (already generated)

### PR DESCRIPTION
**Pull request by dghant1024**
Adding this PR to resolve #91

### CODE REVIEW

File: internal/plugins/workload/v1/scaffolds/templates/readme.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	}
+
+	f.IfExistsAction = machinery.OverwriteFile
+
+	f.TemplateBody = readmefileTemplate
+
+	return nil
+}
+
+// func (p *Readme) Scaffold(fs machinery.Filesystem) error {
```
Comment (scottd018) [inline internal/plugins/workload/v1/scaffolds/templates/readme.go:32]: Removed commented fields

File: internal/plugins/workload/v1/scaffolds/templates/readme.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+// 	return nil
+// }
+
+const readmefileTemplate = `A Kubernetes operator built with
+[operator-builder](https://github.com/vmware-tanzu-labs/operator-builder).
+
+## Local Development & Testing
+
+To install the custom resource/s for this operator:
```
Comment (scottd018) [inline internal/plugins/workload/v1/scaffolds/templates/readme.go:41]: resources

File: internal/plugins/workload/v1/scaffolds/templates/readme.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+bash
+make install
+` + "```" + `
+
+To run the controller locally against a test cluster, make sure you have a
+kubeconfig set up for a test cluster, then run:
+
+    make run
+
+You can then test the operator by creating the sample manifest/s:
```
Comment (scottd018) [inline internal/plugins/workload/v1/scaffolds/templates/readme.go:53]: manifests

File: internal/plugins/workload/v1/scaffolds/templates/readme.go [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+bash
+make install
+make deploy
+` + "```" + `
+
+To clean up:
+
+` + "```" + `
+bash
+make uninstall
```
Comment (scottd018) [inline internal/plugins/workload/v1/scaffolds/templates/readme.go:92]: Remove this line

**Review by scottd018 — CHANGES_REQUESTED** [review]

**Comment by scottd018** [comment]
In addition to small fixes in readme.go file, please fix:

- commit messages to conform with conventional commit format

**Comment by dghant1024** [comment]
Added suggested changes and merged

File: internal/plugins/workload/v1/scaffolds/templates/readme.go (hunk 1 above)
Comment (dghant1024) [reply internal/plugins/workload/v1/scaffolds/templates/readme.go]: done

**Pull request closed without merging**